scripts/local/e2e_test.sh

```diff
@@ -9,6 +9,49 @@ TELEPORTER_PATH=$(
   cd ../.. && pwd
 )
 
+function printHelp() {
+    echo "Usage: ./scripts/local/e2e_test.sh [--component component]"
+    echo ""
+    printUsage
+}
+
+function printUsage() {
+    cat << EOF
+Arguments:
+    --component <component>          Component test suite to run. Must be a valid directory in tests/local/
```

Comment:
Love the change to be able to only test the component that changed but should we still have the ability to run all of them easily?

I'm thinking it might be worthwhile locally and maybe on merge to master which would be a good sanity check and be rarer than each PR commit push

Reply:
For CI, we should run the suites in parallel on separate runners in all scenarios for expediancy and isolation. The local use case makes sense though. I've modified the script to run all components by default if no argument is provided.